pkg/reconciler/revision/controller.go

```diff
@@ -115,7 +116,7 @@ func newControllerWithOptions(
 		logger.Info("Fetch GitHub commit ID from kodata failed", zap.Error(err))
 	}
 
-	digestResolveQueue := workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "digests")
+	digestResolveQueue := workqueue.NewNamedRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(20*time.Millisecond, 30*time.Minute), "digests")
```

Comment:
This drops the bucket rate limiting thing though, I think? Is that intentional?

Reply:
oops no, oversight, added back.